lib/features/settings/views/settings_page.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/material.dart';
+
+class SettingsPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+          child: Center(
+        child: Text("Settingsy"),
+      )),
```

Comment:
never let two )) be next to each other! Always split those with ',' the code will look more organised. Also note that this quadruple space indentation will change to double after that change

Reply:
Ready